commit 1dd301b7c302b9df980946e2d04ca99748754991
Author: Mark Wielaard <mark@klomp.org>
Date:   Mon Jul 17 17:26:25 2017 +0200

    backends: Swap sys/ptrace.h and asm/ptrace.h include order on s390.
    
    glibc 2.26 changed the sys/ptrace.h header so that it cannot be included
    after asm/ptrace.h. We still need to include the kernel asm/ptrace.h for
    the ptrace_area definition. Including it after sys/ptrace.h works against
    both old and new glibc.
    
    Signed-off-by: Mark Wielaard <mark@klomp.org>

diff --git a/backends/s390_initreg.c b/backends/s390_initreg.c
index 011305c..23bf8ed 100644
--- a/backends/s390_initreg.c
+++ b/backends/s390_initreg.c
@@ -34,8 +34,8 @@
 #include <assert.h>
 #if defined(__s390__) && defined(__linux__)
 # include <sys/user.h>
-# include <asm/ptrace.h>
 # include <sys/ptrace.h>
+# include <asm/ptrace.h>
 #endif
 
 #define BACKEND s390_
